lib/manager/common.ts

```diff
@@ -215,9 +215,9 @@ export interface UpdateArtifact {
   config: UpdateArtifactsConfig;
 }
 
-export interface UpdateDependencyConfig {
+export interface UpdateDependencyConfig<T = Record<string, any>> {
```

Comment:
```suggestion
export interface UpdateDependencyConfig<T = Record<string, unknown>> {
```
To be more strict here? Or will it cause to many other required changes. 🤔

Reply:
Yes, too many type errors across managers